tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/newstyle@2023_01_11/stacktrace_threaded_cocoa_below.pysnap

```diff
@@ -0,0 +1,253 @@
+---
+created: '2023-12-12T00:48:13.270368Z'
+creator: sentry
+source: tests/sentry/grouping/test_variants.py
+---
+app:
+  hash: "9592be4d4010972d1fcf6b0907d82458"
+  component:
+    app*
+      threads*
+        stacktrace*
+          frame (non app frame)
+            function (ignored unknown function)
+              "<redacted>"
+          frame*
+            function*
+              "main"
+          frame*
+            function*
+              "unicorn"
+          frame (non app frame)
+            function*
+              "UIApplicationMain"
+          frame (non app frame)
+            function*
+              "-[UIApplication _run]"
+          frame (non app frame)
+            function*
+              "GSEventRunModal"
+          frame (non app frame)
+            function*
+              "CFRunLoopRunSpecific"
+          frame (non app frame)
+            function*
+              "__CFRunLoopRun"
+          frame (non app frame)
+            function*
+              "__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__"
+          frame (marked out of app by stack trace rule (family:native package:/usr/lib/** -app))
+            function*
+              "_dispatch_main_queue_callback_4CF"
+          frame (marked out of app by stack trace rule (family:native package:/usr/lib/** -app))
+            function*
+              "_dispatch_main_queue_drain"
+          frame (marked out of app by stack trace rule (family:native package:/usr/lib/** -app))
+            function*
+              "_dispatch_client_callout"
+          frame (marked out of app by stack trace rule (family:native package:/usr/lib/** -app))
+            function*
+              "_dispatch_block_async_invoke2"
+          frame (non app frame)
+            function*
+              "__NSOQSchedule_f"
+          frame (non app frame)
+            function*
+              "__NSOPERATIONQUEUE_IS_STARTING_AN_OPERATION__"
+          frame (non app frame)
+            function*
+              "-[NSOperation start]"
+          frame (non app frame)
+            function*
+              "__NSOPERATION_IS_INVOKING_MAIN__"
+          frame (non app frame)
+            function*
+              "-[NSBlockOperation main]"
+          frame (non app frame)
+            function*
+              "__NSBLOCKOPERATION_IS_CALLING_OUT_TO_A_BLOCK__"
+          frame (ignored by stack trace rule (category:internals -group))
+            function*
+              "__46+[FudgeGlobalHandler setupGlobalHandlersIfNeeded]_block_invoke_2"
```

Comment:
This is an in-app frame that block_invoke should not mark as ignored.
The next PR should show this being fixed.